bitswap/client/internal/session/session_test.go

```diff
@@ -345,8 +340,6 @@ func TestSessionOnPeersExhausted(t *testing.T) {
 }
 
 func TestSessionFailingToGetFirstBlock(t *testing.T) {
-	test.Flaky(t)
-
```

Comment:
@gammazero  seems to be still flaky on Windows: https://github.com/ipfs/boxo/actions/runs/12167338127/job/33935669596#step:13:256